acquia/acquia_cms
# ACMS-1385: Images added in site studio package.

**Motivation**
Fixes #ACMS-1385

**Proposed changes**
Images added in site studio package.

**Alternatives considered**
NA

**Testing steps**
- Install site with this codebase
- Install site studio and starter module
- `drush in acquia_cms_site_studio acquia_cms_starter -y`
- verify images and icons are rendering.

**Merge requirements**
